Jenkinsfile

```diff
@@ -7,6 +7,10 @@ pipeline {
     stage ('Test') {
       steps {
         script { checkout scm }
+        sh '''
+            echo '[testenv]' >> tox.ini
+            echo 'basepython = python3.9' >> tox.ini
```

Comment:
Can we add this directly in the file?

Reply:
After discussion with @fblackburn1 we decided that this ugly hack was not viable. 
I am creating a specific bullseye runner for this purpose. Plus it will lighten Jenkins community post-upgrade.
As soon as the worker is ready I will remove this part